cdk/domino_cdk/provisioners/efs.py

```diff
@@ -45,7 +51,7 @@ def provision_efs(self, stack_name: str, cfg: config.EFS, vpc: ec2.Vpc, security
             security_group=security_group,
             performance_mode=efs.PerformanceMode.GENERAL_PURPOSE,
             throughput_mode=efs.ThroughputMode.BURSTING,
-            vpc_subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE),
+            vpc_subnets=ec2.SubnetSelection(subnet_type=ec2.SubnetType.PRIVATE_WITH_NAT),
```

Comment:
Did CDK get upgraded on us somehow? Like we failed to pin it correctly? I wouldn't expect there to be any required code changes here.

Reply:
API is being deprecated 
```
Warning:  @aws-cdk/aws-ec2.SubnetType#PRIVATE is deprecated.
  use `PRIVATE_WITH_NAT`
  This API will be removed in the next major release.
Warning:  @aws-cdk/aws-ec2.SubnetType#PRIVATE is deprecated.
  use `PRIVATE_WITH_NAT`
  This API will be removed in the next major release.
```

Reply:
I do remember seeing those at some point earlier, so wfm. Don't mind the change, just wanted to make sure it wasn't masking some other unintended change.